### PR TITLE
change urls to use explorer gochain io

### PR DESCRIPTION
**Pull request by ralyodio**

### CODE REVIEW

File: config/main.json [hunk 1]
```diff
@@ -10,8 +10,8 @@
     { "addr": "0xc6b330df38d6ef288c953f1f2835723531073ce2", "info": "Deployed 07/08/2016 -- OLD, DO NOT USE" }
   ],
   "ethTestnet": false,
-  "ethChainId": 1,
-  "ethProvider": "http://localhost:8545",
+  "ethChainId": 60,
+  "ethProvider": "https://testnet-rpc.gochain.io:443",
```
Comment (jmank88) [inline config/main.json]: ~testnet-~ rpc.gochain.io:443

**Comment by jmank88** [comment]
We might want to move the actual main/test configuration stuff into our internal repo, and only have development stuff in this repo. Nothing internal is being exposed right now, so NBD.

**Comment by chovy** [comment]
Ok, @treeder if you want to create a godelta-config repo that is private we can move the config into that and I'll import into this project.

**Comment by jmank88** [comment]
We already have an internal repo for deployments and configuration. They shouldn't have to know about one another at all. The deployment will only have configuration files and a compose file using images published from this repo. (development should still be stand-alone with this repo only)

**Comment by chovy** [comment]
this godelta app needs to know where the endpoints live, how can you do that without it knowing about the config repo?

**Comment by jmank88** [comment]
Development config should be stubbed out and settable via env vars or something, and default to `localhost`.

**Comment by jmank88** [comment]
If it's extra work to load env vars then that's not necessary for now. I think it's fine to expect someone to modify the file or a copy when they want to point somewhere else.

**Pull request merged**